README.rst

```diff
@@ -1361,6 +1361,30 @@ To use the registry in a ``Dockerfile``:
     ARG DOCKER_REGISTRY
     FROM $DOCKER_REGISTRY/busybox:latest
 
+Docker image build/tag/push race condition
```

Comment:
I would make this "Caution when using the `latest` tag" or something like this, since this is where it occurs most often.